src/components/MermaidDash.js

```diff
@@ -423,18 +422,20 @@ class MermaidDash extends Component {
     return new Set(result).size;
   }
 
-  getUserCount(array, choices) {
-    const projectFilter = array.map(({ project_id }) => project_id);
-    const projectSet = [...new Set(projectFilter)];
+  getUniqueYearCount(array) {
+    const allYearsFromMinMaxRanges = array.reduce((allYears, item) => {
+      let minYear = Number(item.date_min.substring(0, 4));
```

Comment:
This is probably fine, but would it be better to create a `Date()` object and then do `getFullYear()`, in case the format of the date changes in the future?